global.d.ts

```diff
@@ -1,5 +1,5 @@
-import { PrismaClient } from "@prisma/client";
+import { PrismaClient } from '@prisma/client';
 
 declare global {
-    var prisma: PrismaClient;
-}
\ No newline at end of file
+  var prisma: PrismaClient;
```

Comment:
okay I'm noticing a lot of formatting issues here. we can keep these in, but yeah let's just make another ticket in the future to make formatting cleaner

Reply:
hi! yes so I saw David's comment from earlier too, but these were formatting fixes that needed to happen for us to pass the CI/CD checks

Reply:
Mentioned below, but in the future I would just omit these files from git for this PR